src/job-details/JobDetails.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import {
+  Button,
+  Card, CardActions, CardContent, Typography,
+} from '@material-ui/core';
+import { ArrowDropDown } from '@material-ui/icons';
+import { Rating } from '@material-ui/lab';
+import { GET_JOB } from '../sources';
+import './styles.css';
+import CompanyDetails from './CompanyDetails';
+
+const JobDetails = () => {
+  const { id } = useParams();
+  const [companyCardVisibility, setCompanyCardVisibility] = useState(false);
+  const [job, setJob] = useState(null);
```

Comment:
Yet again, any reason to have the job on the state ?

Reply:
fixed